script/cadence/scripts/get-proxy-newmethod.cdc

```diff
@@ -1,8 +1,8 @@
 import FlowColdStorageProxy from 0xProxy
 
-pub fun main(addr: Address): String {
+access(all) fun main(addr: Address): String {
     let acct = getAccount(addr)
-    let ref = acct.getCapability(FlowColdStorageProxy.VaultCapabilityPublicPath).borrow<&FlowColdStorageProxy.Vault>()
+    let ref = acct.capabilities.borrow<&FlowColdStorageProxy.Vault>(FlowColdStorageProxy.VaultCapabilityPublicPath)
     if let vault = ref {
         return vault.newMethod()
```

Comment:
I'm probably missing context - I don't see this on the `Vault` interfaces, what is this for?

Reply:
None of the scripts or code relating to proxies is actually used, these are a holdover for some functionality that had been planned and then stopped.